macros/edr/materializations/model/metrics.sql

```diff
@@ -22,8 +22,20 @@
   {% do return(metrics) %}
 {% endmacro %}
 
-{% macro query_metrics() %}
+{% macro can_query_metrics() %}
   {% if not elementary.get_config_var('collect_metrics') %}
+    {% do return(false) %}
+  {% endif %}
+
+  {% if model.config.require_partition_filter %}
+    {% do return(false) %}
```

Comment:
Wonder if we should send only the `build_timestamp` metric or is it negligible 🤔